astropy/io/fits/tests/test_image.py

```diff
@@ -927,6 +927,7 @@ def test_image_update_header(self):
 
     # The test below raised a `ResourceWarning: unclosed transport` exception
     # due to a bug in Python <=3.10 (cf. cpython#90476)
+    @pytest.mark.slow
```

Comment:
Here too!

Reply:
I note that even cutting 90% of sleep time in this test only makes it 5 times longer than the second slowest in the module. That's about 0.2s on my machine, which is probably acceptable. Let's try !

Reply:
Like I said before, I am not comfortable marking anything in `io.fits` as slow here. I will defer to @saimn .

Reply:
Well there is no good value here, depends on the system, disk, caching etc.
So 1s is maybe a bit too much but seems reasonable to avoid wasting time debugging issues on CI or on exotic platforms.